src/Buerokratt.Common.UnitTests/Dmr/DmrServiceTests.cs

```diff
@@ -14,76 +17,158 @@ namespace Buerokratt.Common.UnitTests.Dmr
 {
     public sealed class DmrServiceTests : IDisposable
     {
-        private static readonly DmrServiceSettings DefaultServiceConfig = new()
+        private readonly MockHttpMessageHandler _httpMessageHandler = new();
+        private readonly Mock<ILogger<DmrService>> _logger;
+
+        public DmrServiceTests()
         {
-            DmrApiUri = new Uri("https://dmr.fakeurl.com")
-        };
+            _logger = new Mock<ILogger<DmrService>>();
+            _ = _logger
+                .Setup(m => m.IsEnabled(It.IsAny<LogLevel>()))
+                .Returns(true);
+        }
 
-        private readonly MockHttpMessageHandler httpMessageHandler = new();
-        private readonly Mock<ILogger<DmrService>> logger = new();
+        private static ICentOpsService ConfigureCentOps()
+        {
+            var centOpsMock = new Mock<ICentOpsService>();
+            _ = centOpsMock.Setup(x => x.FetchParticipantsByType(ParticipantType.Dmr))
+                       .ReturnsAsync(new[]
+                       {
+                           new Participant
+                           {
+                               Host = "https://dmr.fakeurl.com",
```

Comment:
Should/could this be a setting? Are there any scenarios where we'll need to set this to a different value?